packages/frontend/mobile/src/components/form-cocktail/NamePart.tsx

```diff
@@ -0,0 +1,138 @@
+import { Ionicons } from '@expo/vector-icons';
+import {
+  StyleSheet,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View,
+} from 'react-native';
+import Svg, { G, Path } from 'react-native-svg';
```

Comment:
`Unable to resolve "react-native-svg" from "src/components/form-cocktail/NamePart.tsx"`

Reply:
ok, done : bun add react-native-svg